Code/src/main/java/nl/utwente/viskell/haskell/env/CatalogFunction.java

```diff
@@ -43,4 +47,8 @@ public final int compareTo(final CatalogFunction entry) {
         return this.getName().compareTo(entry.getName());
     }
 
+    public boolean getConstructor() {
```

Comment:
Would prefer the getter method to be called `isConstructor` per JavaBeans conventions.